Swahilipot-Hub-Developers/industrial-attachment
# Add New Images and Gallery Page Functionality

## Description

This pull request introduces a new gallery page to the project, showcasing additional images that highlight various events and activities. The new images enhance the visual appeal and content of the application, providing users with a more engaging experience.

Changes made:
- Created a new Gallery page component.
- Added new images with descriptions for better context.
- Improved layout to ensure the images are displayed responsively.

## Type of change/addition


## How Has This Been Tested?

I tested the gallery functionality by navigating to the new Gallery page and verifying that all images load correctly with their corresponding descriptions. 

Instructions to reproduce the tests:
1. Navigate to the Gallery page from the homepage.
2. Verify that all images are displayed without errors.
3. Check that the descriptions are accurate and formatted correctly.


## Checklist:


## Screenshots (if applicable):

## Gallery Screenshot
<img width="1440" alt="Screenshot 2024-10-26 at 20 24 25" src="https://github.com/user-attachments/assets/b4fa0fe6-9a0c-4e6b-bdd1-429dc4c42d1d">
